src/main/java/alfio/controller/api/v2/user/ReservationApiV2Controller.java

```diff
@@ -709,6 +737,90 @@ public ResponseEntity<Boolean> removeCode(@PathVariable String reservationId, @R
         return ResponseEntity.ok(res);
     }
 
+    @GetMapping("/reservation/{reservationId}/get-applicable-custom-payment-method-details")
```

Comment:
I'd suggest to remove "get" from the path, as it might be redundant.
should be: `/reservation/{reservationId}/applicable-custom-payment-method-details`